### DISCUSSION TITLE
Webvault Design in 1.31.0

### DISCUSSION THREAD

**Discussion by bokkabonga** [dani-garcia/vaultwarden#4722] [General]
I just updated to the latest release. The new Webvault (as seen in the official bitwarden server) has been changed to have a black Background color.

Is it possible to change that color back to the "bitwarden" blue? Maybe as an Env-Variable?

**Comment by BlackDex** [comment]
No, it's not possible to have this changed via an env variable. Those items are fixed during the build and not that easy to change at runtime.

We might change the color to something else in the future.
But we do want to have it a bit different then the default color of Bitwarden to make it a bit more clear it is an other version.

**Reply by bokkabonga** [reply]
Makes sense to me. 

Even though i think that the different logo in the top left corner should be distinctive enough.
